packages/react-components/components/select/src/select.jsx

```diff
@@ -21,6 +21,10 @@ const propTypes = {
      * A select can have a list of actions after his items.
      */
     actions: arrayOf(shape(ACTION_SHAPE)),
+    /**
+     * @ignore
+     */
+    multiple: bool,
```

Comment:
You dont need to add `multiple` here since you don't use it, it would be provided to the select component through `...rest`.